### ISSUE TITLE
Poczet WarsawJS

### ISSUE THREAD

**Issue by rkaw92** [warsawjs/incubator#1]
# Zarys
Jako że WarsawJS to nie tylko wydarzenie, ale przede wszysktim społeczność, proponuję utworzenie **pocztu** - czegoś w rodzaju listingu osób i firm z Warszawy powiązanych z JavaScriptem. To byłoby miejsce, gdzie można przedstawić siebie lub swoją organizację, i krótko opowiedzieć, gdzie wykorzystuje się JS, a także co fajnego się stworzyło (linki do repo, usługi w cloudzie).

# Korzyści
* Przedstawienie i promocja osób/organizacji zainteresowanych tematem
* Pokazanie Warszawy jako obfitującej w JavaScript :smile: 
* Wymiana doświadczeń (wiadomo, kogo "łapać" na meetupie, żeby porozmawiać o danych zagadnieniach)
* Przegląd zastosowań różnych technologii związanych z JS (front-end, Node, coś a'la "popularity contest")
* Zwiększenie zaangażowania stałych i nowych uczestników WarsawJS w rozwój nowych pomysłów przez oficjalne włączenie ich w "poczet WarsawJS"!

To nie byłaby ani grupa facebookowa, ani platforma chatu / issue tracker, tylko faktyczny zbiór "wizytówek" - dyskusją na konkretne tematy można by się zajmować np. na slacku.

**Comment by piecioshka** [comment]
Dzięki @rkaw92 za pierwszy pomysł! Keep going! 💪

> Jako że WarsawJS to nie tylko wydarzenie, ale przede wszystkim społeczność

W 100% się zgadzam. WarsawJS to przede wszystkim **społeczność**!

> czegoś w rodzaju listingu osób i firm z Warszawy powiązanych z JavaScriptem. 

* Listing **osób**?
To podchodzi pod zbieranie danych osobowych, a to z kolei trzeba zgłosić do GIODO.

* Listing **firm**?
Odpada w przed biegach, ponieważ jako organizacja promujemy tylko naszych sponsorów. Muszą oni czymś się różnić od innych firm, w których też się pisze w JavaScript, ale nie są sponsorami WarsawJS.

> To byłoby miejsce, gdzie można przedstawić siebie lub swoją organizację, i krótko opowiedzieć, gdzie wykorzystuje się JS, a także co fajnego się stworzyło

Domyślam się, że chodzi Ci o stronę internetową np. https://warsawjs.com/vcards, gdzie widziałbyś wizytówki wszystkich zainteresowanych istnienia w poczcie?

**Comment by rkaw92** [comment]
Tak, właśnie jako stronę, gdzie można opublikować swój krótki profil. Faktycznie, aspekt ochrony danych osobowych nie przyszedł mi do głowy, jako że wszystkie ze zbieranych danych zasadniczo byłyby podawane do publicznej wiadomości. Może jednak warto to z kimś skonsultować, ponieważ sam kształt danych pasuje do definicji danych osobowych (w szczególności, imię, nazwisko i zdjęcie profilowe pozwalają kogoś rozpoznać). Jestem ciekaw, czy GIODO udziela porad w takich sprawach. To możliwe.

Co do listingu firm, zgadzam się, że sponsorzy powinni być wyróżnieni. Nie wiem tylko, jak to się ma do planów dot. fundacji, ale na razie wycofuję się z części o firmach, ponieważ jest znacząco różna od "wizytówek" osób, które spotykamy na meetupie.
